src/index.spec.ts

```diff
@@ -1,12 +1,180 @@
+import { valid } from 'joi';
 import {
   isInteger,
   toLowerCase,
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
+import fetch from 'node-fetch';
+import { mocked } from 'ts-jest/utils';
 
-test('Testing time!', () => {
-  console.log('welcome');
+ describe('isInteger', () =>{
+  test('Return true if we pass a number', () =>{
+    expect(isInteger(8)).toBe(true);
+  });
+
+  test('Return false if we pass a string', () =>{
+    expect(isInteger('hola')).toBe(false);
+  });
+
+  test('Return false if we pass a null value', () =>{
+    expect(isInteger(null)).toBe(false);
+  });
+
+  test('Return true if we pass a negative number', () =>{
+    expect(isInteger(-6)).toBe(true);
+  });
+
+  test('Return false if we pass NaN', () =>{
+    expect(isInteger(NaN)).toBe(false);
+  });
+});
+
+describe('toLowerCase', () =>{
+  const defaultMessage = 'Please provide a string'
+
+  test('Return default message if we dont pass a string', () =>{
+    expect(toLowerCase('')).toBe(defaultMessage);
+  });
+
+  test('Return default message if we pass null', () =>{
+    expect(toLowerCase(null)).toBe(defaultMessage);
+  });
+
+  test('Return lowercase if we pass capital letters', () =>{
+    expect(toLowerCase('THIS IS A TEST')).toEqual('this is a test');
+  });
+  
+  test('Return lowercase if we pass lowercase string', () => {
+    expect(toLowerCase('this is a test')).toEqual('this is a test')
+  })
+});
+
+describe('removeDuplicatesFromArray', () => {
+  const duplicateItemsArray = ['@', '@', 2, 2];
+  const uniqueItemsArray = ['this', 'is', 'a', 'test', 1, 2, 3];
+  const singleItemArray = [1];
+  const emptyArray = [];
+  const hashValue = {key: 'value'};
+  const nullValue = null
+
+  test('Return removed items array if we pass an array with duplicate items', () => {
+    expect(removeDuplicatesFromArray(duplicateItemsArray)).toEqual(['@', 2]);
+  });
+
+  test('Return same array if we pass an array with unique items', () => {
+    expect(removeDuplicatesFromArray(uniqueItemsArray)).toEqual(uniqueItemsArray);
+  });
+
+  test('Return empty array if we pass an empty array', () => {
+    expect(removeDuplicatesFromArray(emptyArray)).toEqual(emptyArray);
+  });
+  
+  test('Return same array if we pass an array with a single item', () => {
+    expect(removeDuplicatesFromArray(singleItemArray)).toEqual(singleItemArray);
+  });
+
+  test('Throw an error when pass invalid value', () => {
+    function passEmptyArray(){
+      removeDuplicatesFromArray(hashValue as any);
+    };
```

Comment:
Why are you creating a wrapper of your testing function?

Avoid this, don't create wrappers of your testing functions. And avoid use `any` as well